internal/auth/oidc/service_token_request_test.go

```diff
@@ -234,7 +234,7 @@ func Test_TokenRequest(t *testing.T) {
 			atRepoFn:     atRepoFn,
 			authMethodId: testAuthMethod.PublicId,
 			tokenRequest: func() string {
-				exp, err := ptypes.TimestampProto(time.Now().Add(AttemptExpiration).Truncate(time.Second))
+				exp := timestamppb.New(time.Now().Add(AttemptExpiration).Truncate(time.Second))
 				require.NoError(t, err)
```

Comment:
You can remove the NoError check following both of these timestamppb.New() calls here and elsewhere in this PR.

Reply:
ty